<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagesManage</class>
 <widget class="QWidget" name="ImagesManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33);</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>361</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
QListWidget
{
	background-color: rgb(33, 33, 33);
	color: rgb(180, 180, 180);
}
/**列表项*/
QListWidget::item
{
    color:rgb(200,200,200);
    border:0px solid gray;
}
/**列表项扫过*/
QListWidget::item:hover
{
    color:rgb(210,210,210);
	background-color: rgb(50, 50, 50);
}
/**列表项选中*/
QListWidget::item::selected:active
{ 
    color:rgb(255,200,0);
	background-color: rgb(50, 50, 50);
}
QListWidget::item:selected
{
    color:rgb(255,200,0);
	background-color: rgb(50, 50, 50);
}
</string>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bn_image">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
	margin:4px 0px 0px 0px
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bn_add">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}

QPushButton:!enabled{
	color: rgb(100, 100, 100)
}</string>
   </property>
   <property name="text">
    <string>新增</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bn_close">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}

QPushButton:!enabled{
	color: rgb(100, 100, 100)
}</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bn_choose">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}

QPushButton:!enabled{
	color: rgb(100, 100, 100)
}</string>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bn_delete">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}

QPushButton:!enabled{
	color: rgb(100, 100, 100)
}</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
